docs/src/pages/[platform]/getting-started/usage/nextjs/examples/UseClientAdvancedExampleCode.tsx

```diff
@@ -0,0 +1,209 @@
+'use client';
```

Comment:
Same file as existing `AdvancedExampleCode.tsx`, but with `use client`. 

I thought about just reusing the same `tsx` file and describe separately that you should add `'use client'` to the top, but I thought it'd be easier for customers to read if we put everything in this one file.